SonarQube.Client.Tests/SonarQube.Client.Tests.2015.csproj

```diff
@@ -0,0 +1,143 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Forge a VS2015 version of the test project that will be used by SonarLint for VS.